banana.js

```diff
@@ -75,8 +75,8 @@ banana.reversed = (lang) => {
   return reversed.join("");
 };
 
-banana.peel   = () => "🍌";
-banana.cousin = () => "🍆";
+banana.peel   = () => "🍌🍌🍌";
```

Comment:
Please take an optional parameter for number of bananas, defaulting to 1, to avoid breaking compat